content/blog/2015-04-23-prometheus-posts-around-the-web.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Prometheus posts around the web"
```

Comment:
A blog post title is an appropriate time to use [title case](http://www.grammar-monster.com/lessons/capital_letters_title_case.htm) for once: "Prometheus Posts around the Web".